docs/visualization/plotly/showcase/PdosPlot.ipynb

```diff
@@ -43,7 +44,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "plot = sisl.get_sile(\"../files/SrTiO3.PDOS\").plot(Erange=[-10,10])"
+    "plot = sisl.get_sile(\"../../../../files/tests/sisl/io/siesta/SrTiO3.PDOS\").plot(Erange=[-10,10])"
```

Comment:
this should be changed to the `env-var` that points to the `SISL_FILES_TESTS`, if not defined, then not possible.